src/common/models/measure/measure.mocha.ts

```diff
@@ -178,4 +197,58 @@ describe('Measure', () => {
     });
 
   });
+
+  describe('toApplyExpression', () => {
+
+    describe('no transformation', () => {
```

Comment:
could we use dynamically generated tests? all three tests are almost identical
https://mochajs.org/#dynamically-generating-tests

Reply:
Sure.
Funny thing I have been thinking about it when I was creating the tests ;-)